gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.ts

```diff
@@ -71,4 +71,7 @@ export class ApiDocumentationV4PagesListComponent implements OnInit, OnChanges {
   getLogoForPageType(pageType: PageType) {
     return `assets/logo_${pageType.toLowerCase()}.svg`;
   }
+  getTooltipForPageType(pageType: PageType) {
+    return pageType.toLowerCase();
```

Comment:
What do you think about providing formatting? --> Markdown, AsyncAPI, Swagger, AsciiDoc, etc.

Reply:
I was wondering if it would be necessary, but yes I maybe for better readability...